src/nupic/algorithms/Connections.hpp

```diff
@@ -595,6 +585,10 @@ class Connections : public Serializable
   Segment nextSegmentOrdinal_;
   Synapse nextSynapseOrdinal_;
 
+  bool timeseries_;
```

Comment:
nit, could put these three members in a struct to make the variables more clearly grouped.

Reply:
I added a comment instead which explains that they're all part of the same functionality.